.travis.yml

```diff
@@ -42,5 +42,4 @@ before_install:
   - if [[ ! $GITHUB_TOKEN ]]; then echo "no github token"; fi
 
 after_script:
-  - if [[ $TRAVIS_PHP_VERSION = '5.6' ]]; then php vendor/bin/coveralls -v ; fi
```

Comment:
It's not a big problem to keep it, I just saw on https://coveralls.io/github/doctrine/migrations that the last build was March 2015 so I assumed it was useless here as well